source/playbooks/reusing-and-sharing-playbooks.rst

```diff
@@ -0,0 +1,13 @@
+Reusing and sharing playbooks
+=============================
```

Comment:
Can you add the appropriate subscription and deploy badges to this page, please? Also change "Reusing and sharing playbooks" to "Reuse and share playbooks" to remove the gerund.